### DISCUSSION TITLE
Why is this? Dark theme studio follower count seems harder to see compared to light theme

### DISCUSSION THREAD

**Discussion by Endermaster3690** [ScratchAddons/ScratchAddons#1804] [Help]
So the studio I am in, Games of Glory, has 48 followers. However, the follower count, it seemed a bit harder to see in dark theme to me. i went in light theme and it was completely good i think.

Dark theme:
![Screenshot 2021-03-10 135600](https://user-images.githubusercontent.com/80355301/110689902-e0be7880-81a8-11eb-809c-a729faab8c49.png)

Light theme:
![3](https://user-images.githubusercontent.com/80355301/110689933-e87e1d00-81a8-11eb-9ae4-14a585c41fc4.png)

I don't know why.

**Comment by mxmou** [comment]
Most gray text isn't currently changed by the dark theme. This also includes some numbers on the My Stuff page.

It would be a good idea to change this in the future, probably when implementing #645.

**Reply by Endermaster3690** [reply]
hm
